couler/argo.py

```diff
@@ -242,10 +242,16 @@ def create_s3_artifact(
     )
 
 
-def create_secret(secret_data, namespace="default", name=None, dry_run=False):
+def create_secret(
+    secret_data, namespace="default", name=None, dry_run=False, exists=False
+):
     """Store the input dict as a secret in k8s, and return the secret name."""
     secret = Secret(
-        namespace=namespace, data=secret_data, name=name, dry_run=dry_run
+        namespace=namespace,
+        data=secret_data,
+        name=name,
+        dry_run=dry_run,
+        exists=exists,
```

Comment:
I am not sure if this is a good arg to add for this function as this function is for creating new secrets

Reply:
Would you recommend creating a new function, accessing the secret through env then?
To the same place where the create_secret() is. Any name preferences to it?

- Noora